phpstan-baseline.neon

```diff
@@ -1,15 +1,5 @@
 parameters:
 	ignoreErrors:
-		-
-			message: "#^Negated boolean expression is always false\\.$#"
```

Comment:
do we have more errors you consider a false-positive in this file?
I can have a look into improving the phpstan side of things

Reply:
I created issue #227 for the last item in phpstan ignoreErrrors.